src/Kernel-Tests/IntegerTest.class.st

```diff
@@ -382,6 +382,12 @@ IntegerTest >> testEven [
 	
 ]
 
+{ #category : #'tests - bitLogic' }
+IntegerTest >> testFactorial [	
```

Comment:
I think it is better to confider `factorial` as the default implementation. So instead of asserting if `factorial` is equal to `slowFactorial`, I think it is better to have a normal test here, and then in `testSlowFactorial`, we can assert whether `slowFactorial` is equal to `factorial`. This way we would reverse the dependency and it will be `slowFactorial` that depends on `factorial` and not the other way around. It is good because `slowFactorial` could be removed or renamed in the future.